stock_picking_auto_create_lot_qty/tests/common.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (C) 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+class CommonStockPickingAutoCreateLotQty(object):
```

Comment:
Why would inheriting from `object`?
Class can inheriting from SavepointCase.

Reply:
I'll fix it